Sources/Nitrozen-SwiftUI/Elements/Avatar/Avatar+Appearance.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  File.swift
+//  
+//
+//  Created by Keyur Patel on 18/04/23.
+//
+
+import Foundation
+
+public extension NitrozenAppearance {
+    
+    class Avatar {
+        
+        var borderWidth: CGFloat
+        var borderColor: SystemColor
+        var backgroundColor: SystemColor
+        var textStyle: TextLabel
+        var disableOpacity: CGFloat
```

Comment:
do we have any logic for `disableOpacity` apart from setting opacity only??
if no other logic, we can remove that from here, so user can directly set from outside